person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(id, age, name = 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+end
+
+private
+
+def of_age?
+  @age >= 18
+end
+
+public
+
+def can_use_services?
+  of_age?
```

Comment:
- We could make this like `of_age?() || @parent_permission` since we're actually checking if they're more than 18 or below 18 years old.